packages/plugin-npm-cli/sources/commands/npm/login.ts

```diff
@@ -88,7 +88,21 @@ export async function getRegistry({scope, publish, configuration, cwd}: {scope?:
   return npmConfigUtils.getDefaultRegistry({configuration});
 }
 
-async function setAuthToken(registry: string, npmAuthToken: string, {configuration}: {configuration: Configuration}) {
+async function setAuthToken(registry: string, npmAuthToken: string, {configuration, scope}: {configuration: Configuration, scope?: string}) {
+  if (scope) {
+    return await Configuration.updateHomeConfiguration({
+      npmScopes: (scopes: {[key: string]: any} = {}) => ({
+        ...scopes,
+        [scope]: {
+          ...scopes[scope],
+          npmRegistryServer: registry,
+          npmPublishRegistry: registry,
```

Comment:
Is that necessary? If we are successfully authenticated it means that the scope registry is already well configured. I feel like it would make more sense to set the token and leave the other fields as they are.

Reply:
Probably not, was uncertain so I set them just to be safe. Removed :+1: